server/models.py

```diff
@@ -112,12 +113,23 @@ class StringList(types.TypeDecorator):
 
     def process_bind_param(self, string_list, dialect):
         # Python -> SQL
-        return ' '.join(string_list)
+        items = []
+        for item in string_list:
+            if " " in item:
+                items.append('"{}"'.format(item))
+            else:
+                items.append(item)
+        return ' '.join(items)
 
     def process_result_value(self, value, dialect):
         # SQL -> Python
-        return value.split()
-
+        values = []
+        for val in shlex.split(value):
```

Comment:
This will break if the value e.g. ends in a backslash (`foo \`) or if it has other characters that have meaning to a shell. I don't have a better way though so can you add a comment that this is fragile?